src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface FruitShopService {
+    void processOfOperations(List<FruitTransaction> fruitTransactions);

```

Comment:
minor: `processOfOperations` sounds more like a noun than a verb to me. `processTransactions` sounds more descriptive.